packages/keyring-controller/src/KeyringController.ts

```diff
@@ -429,7 +467,7 @@ export class KeyringController extends BaseControllerV2<
    * @param account - An account address.
    * @returns Promise resolving to keyring of the `account` if one exists.
    */
-  async getKeyringForAccount(account: string): Promise<unknown> {
+  async getKeyringForAccount(account: string): Promise<Keyring<Json>> {
```

Comment:
Perhaps we should leave these as `unknown` until we migrate each of our keyrings? Otherwise this could be misleading when people use it with the keyrings that don't yet use this interface